site/docs/v1/tech/core-concepts/roadmap.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: doc
+title: Roadmap
+description: The FusionAuth roadmap process
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+FusionAuth is under active development. We often get questions from users and clients asking about when a particular feature will be implemented or can be expected. 
+
+> "Alas, it is always dangerous to prophesy, particularly about the future." - Danish proverb
+
+We love our community and a large part of our development effort is directed by feedback from the community. We also love our paying customers and want to make sure their needs are met as well.
+
+=== Viewing the Roadmap
+
+If you're a paying customer and need delivery timelines for specific features, please https://account.fusionauth.io/account/support/[open a support ticket] with your questions. We'll get back to you with answers.
+
+If you're a community member, it is best to review the https://github.com/FusionAuth/fusionauth-issues/projects/2[issues project] which provides a short term roadmap. 
+
+You can sort the full issues list by https://github.com/fusionauth/fusionauth-issues/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc[the number of votes] which is a good indicator of future development efforts. 
+
+It's important to note that **our roadmap is fluid**. Our development efforts are responsive both to community and customer needs. When you view the lists above, please take them as general guides and not hard and fast commitments. 
+
+=== Giving Feedback
+
+If you're a paying customer and need a particular feature, please https://account.fusionauth.io/account/support/[open a support ticket]. We're happy to do our best to prioritize such requests.
+
+If you're a community member, please https://github.com/fusionauth/fusionauth-issues[vote on issues] with a thumbs up. As mentioned above, we actively incorporate the number of community votes when planning development efforts. 
+
+You can also https://github.com/fusionauth/fusionauth-issues/issues[file feature requests]; please be as detailed as possible about the problem, use cases, and desired solutions.
+
+=== Bugs 
+
+Every FusionAuth release has both features and bug fixes. We appreciate https://github.com/fusionauth/fusionauth-issues/issues[detailed bug reports] with replication steps, version information, log output and as much information as possible. After all, it's difficult to fix a bug without being able to reproduce it.
+
+We prioritize bug fixes based on the impact of the issue, how many users are affected, estimated effort and other factors.
+
+=== If You Require a Feature
+
+Sometimes you just can't wait for a feature to be finished through the normal development process! We get it. 
+
+If you'd like to pay us to build a feature, please link:/contact[contact us] and ask about professional services.
```

Comment:
I don't want to limit inbound requests here, but we may want to add some words here that indicates professional services are not available (with some exceptions) for customers using the Community edition of FusionAuth.

Reply:
Are the exceptions clearly defined? 

If not, rather than limit the inbound requests right now, I'd prefer to use this language. If we get a flood (or even a trickle) of inbound requests that waste Steven's time, then I think we could revise this to indicate the criteria for proserv.